src/main/resources/application.properties

```diff
@@ -106,3 +106,7 @@ rbs.enabled=false
 rbs.poolId=datarepo_v1
 rbs.instanceUrl=https://buffer.tools.integ.envs.broadinstitute.org
 rbs.clientCredentialFilePath=/tmp/buffer-client-sa-account.json
+usermetrics.appId=datarepo
+usermetrics.bardBasePath=
+usermetrics.metricsReportingPoolSize=10
+usermetrics.ignorePaths=/api/repository/v1/jobs/*,/api/repository/v1/configs*,/api/repository/v1/upgrade,/actuator/*,/configuration,/status
```

Comment:
Do we want to include /shutdown in this list?

Reply:
Ah good point.  I might add it in a followon (I suspect we'll iterate on these). It actually won't get logged since the logging only captures authorized calls and there's no way to call shutdown (since we block it at the proxy level)